src/amdgpu.cpp

```diff
@@ -24,19 +24,21 @@ bool amdgpu_verify_metrics(const std::string& path)
         return false;
     }
 
-    switch (header.structure_size)
-    {
-        case 80: // v1_0, not naturally aligned
-			return false;
-        case 96: // v1_1
-        case 104: // v1_2
-        case sizeof(gpu_metrics_v1_3): // v2.0, v2.1
-        case sizeof(gpu_metrics_v2_2):
-            if (header.format_revision == 1 || header.format_revision == 2)
-                return true;
-        default:
-            break;
-    }
+	switch (header.format_revision)
+	{
+		case 1: // v1_1, v1_2, v1_3
+			if(header.content_revision<=0 || header.content_revision>3)// v1_0, not naturally aligned
+				break;
+			cpuStats.cpu_type = "GPU";
+			return true;
+		case 2: // v2_1, v2_2, v2_3
+			if(header.content_revision<=0 || header.content_revision>3)// v2_0, not naturally aligned
```

Comment:
> not naturally aligned

copy/paste error?

Reply:
according to the source code:

https://elixir.bootlin.com/linux/latest/source/drivers/gpu/drm/amd/include/kgd_pp_interface.h#L672

both v1_0 and v2_0 are not naturally aligned and not recommended.